Scroll up on upload

VanillaSixtySix opened the pull request:
This commit remedies a bug where the user's scroll position stays out of bounds after hitting upload. The issue (only tested on Firefox) is caused by the video metadata editor having their elements hidden from view, and the browser not keeping up with that change.

I don't particularly like this fix, but I've barely been able to replicate the issue since.

VanillaSixtySix committed:
Scroll up on upload

This commit remedies a bug where the user's scroll position stays out of bounds after hitting upload.

Sollace commented:
This is actually a bug with firefox and how it handles 3d transforms and scrolling.
I have no idea when they intend to fix this, though. :/

Sollace commented on app/assets/javascripts/components/uploader.js:
```diff
@@ -72,6 +72,7 @@ function Uploader() {
   this.form.addEventListener('submit', event => {
     if (!uploadingQueue) uploadingQueue = new UploadQueue();
     uploadingQueue.enqueue(this);
+    document.querySelectorAll('.tabs')[0].scrollIntoView();
```
You can replace this with `document.querySelector()` to get only the first one.

VanillaSixtySix committed:
Use querySelector over querySelectorAll

This commit is in response to a review stating that querySelector can be used over querySelectorAll(...)[0] to get the first element.
